lib/overcommit/hook/pre_commit/w3c_css_validator.rb

```diff
@@ -0,0 +1,74 @@
+module Overcommit::Hook::PreCommit
+  # Runs `w3c_validators` against any modified CSS files.
+  class W3cCssValidator < Base
+    def run
+      begin
+        require 'w3c_validators'
+      rescue LoadError
+        return :warn, 'w3c_validators not installed -- run `gem install w3c_validators`'
```

Comment:
This should be a failure condition (i.e. return `:fail`). Any other hook that is missing its required executables currently fails.